tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "moduleResolution": "node",
+    "lib": ["es6"],
+    "module": "commonjs",
+    "baseUrl": ".", 
+    "resolveJsonModule": true,
+    "allowJs": true, 
+    "outDir": "build",
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "noImplicitAny": true,
+    "skipLibCheck": true
+  },
+    "exclude":[
+    "./node_modules"
```

Comment:
Please fix spaces, you can add prettier

Reply:
Also, you need a prettier file for configuration because you have a problem with spaces anyway in json file.

Reply:
yes, you're right, sorry, it was disabled in my settings for json files

added config file for prettier